tests/kw_config_loader_test.sh

```diff
@@ -138,4 +139,34 @@ $PWD/$CONFIG_FILENAME"
   assertTrue "Wrong config file reading order.\n$expected_vs_got" $?
 }
 
+function test_show_variables() {
+  local showed_options
+
+  # get all assigned options, including commented ones
+  # remove #'s and ='s to get option names
+  local -A possible_options
+  for o in $(cat "etc/kworkflow_template.config" \
```

Comment:
For the test, use files from `tests/samples/`. Feel free to update it if you need.

Reply:
Makes sense... but it seems like this test is made for developers not to forget to update `kw vars` when they add a new option to `kworkflow_template.config`. Making a copy of it in `tests/samples` would defeat this purpose... What do you think?

Reply:
That's a good point, you're right.

Reply:
For example, right now there is an old version of `kworkflow_template.config` in the `tests/samples` folder.